features/atomic/atomic_mount_unmount.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Atomic mount and unmount test for new upgrade tree
+    Describes tests for mount/umount to new ostree
+
+Background: Atomic hosts are discovered
+      Given "cihosts" hosts from dynamic inventory
```

Comment:
I would recommend changing all instances of `cihosts` to just `all`  since these tests are focused on a single Atomic host.